src/AutoRest.CSharp.V3/Output/Models/Types/ObjectType.cs

```diff
@@ -361,8 +361,8 @@ private IEnumerable<ObjectTypeProperty> BuildProperties()
             // WORKAROUND: https://github.com/Azure/autorest.modelerfour/issues/261
             var existingProperties = EnumerateHierarchy()
                 .Skip(1)
-                .SelectMany(type => type._objectSchema.Properties)
-                .Select(p => p.Language.Default.Name)
+                .SelectMany(type => type.Properties)
```

Comment:
Use ObjectType.Properties here because they were already processed and combined to handle multiple inheritance while schemas were not.